.github/workflows/Jest.yml

```diff
@@ -0,0 +1,25 @@
+name: Jest
```

Comment:
we already have jest in ci, why did you add it again?

Reply:
I just merged the code from the main to mine. I didn't add anything outside of the home screen code and sign-in/sign-up test.